### DISCUSSION TITLE
building=garage ?

### DISCUSSION THREAD

**Discussion by aftocext** [zlant/parking-lanes#102] [Ideas]
Hello
 Very interested by your work !
 
 In my city "Fontaine 38600, in France", I'm working on such theme : parkings
 
 The idea is to have a proper inventory all of space necessary for cars, either public space, or even private space

 You can se [my inventory in "Fontaine 38600, France" here](https://overpass-turbo.eu/s/1fYd) : it is basic as compared with [your work](https://zlant.github.io/parking-lanes/#16/45.1939/5.6905), and I m just starting using "parking_lane" tag, inspired by your work
 
  I was wondering if it were possible to add in your [parking lanes map](https://zlant.github.io/parking-lanes) a specific representation for the  "building=garage" (there are quite a lot  : [see them in "Fontaine 38600, France" here](https://overpass-turbo.eu/s/1fTI) : I could even, may be, propose a PR myself if you offer me some guidance ?
  
 Thank you very much in advance 
  
  Paul
  See  [OSM contributor page](https://wiki.openstreetmap.org/wiki/User:Barnes38)

**Comment by zlant** [comment]
Hello! I think garages are not suitable for this map. Here the focus is on public parking and their availability. And garages are always private. For public garages, need to add a parking tag.

You may add garages in fork.
Here overpass query: https://github.com/zlant/parking-lanes/blob/master/src/parking/data-url.ts#L36
Here need process "building=garage" as"amenity=parking": https://github.com/zlant/parking-lanes/blob/master/src/parking/interface.ts#L186
